src/qt/transactionview.cpp

```diff
@@ -592,6 +591,25 @@ void TransactionView::focusTransaction(const QModelIndex &idx)
     transactionView->setFocus();
 }
 
+void TransactionView::focusTransaction(const uint256& txid)
+{
+    if(!transactionProxyModel)
+        return;
+
+    const QModelIndexList results = this->model->getTransactionTableModel()->match(
+        this->model->getTransactionTableModel()->index(0,0),
+        TransactionTableModel::TxHashRole,
+        QString::fromStdString(txid.ToString()), -1);
+
+    transactionView->setFocus();
+    transactionView->selectionModel()->clearSelection();
+    for (const QModelIndex& index : results) {
```

Comment:
We could ensure first row is visible:
```cpp
for (...) {
    if (index == results[0]) transactionView->scrollTo(index);
```

Reply:
That worked (using `transactionProxyModel->mapFromSource`) when transactions are ordered by date. Unfortunately when they're ordered in reverse it doesn't scroll all the way to the bottom. Maybe we need to wait a little bit for `transactionView` to become aware of its new size?